src/sql/workbench/contrib/assessment/browser/media/detailview.css

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.detailView-toggle {
+	display: inline-block;
+	cursor: pointer;
+}
+
+.dynamic-cell-detail {
```

Comment:
This needs to be scoped down - it shouldn't apply to all cells (same goes for the rest of the rules in this stylesheet)